src/test/java/uk/gov/cslearning/record/repository/InviteRepositoryTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.cslearning.record.repository;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.cslearning.record.domain.Invite;
+
+import javax.transaction.Transactional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.notNullValue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class InviteRepositoryTest {
+
+    @Autowired
+    private InviteRepository inviteRepository;
+
+    @Test
+    public void shouldSaveInvite() {
```

Comment:
Can you add a test here for the relationship between the invite and the event. It would also be good to test the findById method as in the other Entity classes the id was changed to an int but the repository was still expecting a long.